qa/L0_infer/infer_test.py

```diff
@@ -39,8 +39,8 @@
     int(os.environ.get('TEST_SYSTEM_SHARED_MEMORY', 0)))
 TEST_CUDA_SHARED_MEMORY = bool(int(os.environ.get('TEST_CUDA_SHARED_MEMORY',
                                                   0)))
-CPU_ONLY = (os.environ.get('TRITON_SERVER_CPU_ONLY') is not None)
-
+#CPU_ONLY = (os.environ.get('TRITON_SERVER_CPU_ONLY') is not None)
+CPU_ONLY = False
```

Comment:
Why you need to change the test? Won't that break existing tests?

Reply:
Sorry this is to get the unit test run properly. I'll change this back